<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="0" colspan="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Orientation reflex 2:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Orientation reflex 1:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Lattice constants:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sample name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="DeviceParamEdit" name="samplenameEdit">
       <property name="dev" stdset="0">
        <string>Sample</string>
       </property>
       <property name="param" stdset="0">
        <string>samplename</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Space group:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Psi0 (sample rotation) offset:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Lattice angles:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="DeviceParamEdit" name="latticeEdit">
       <property name="dev" stdset="0">
        <string>Sample</string>
       </property>
       <property name="param" stdset="0">
        <string>lattice</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="DeviceParamEdit" name="anglesEdit">
       <property name="dev" stdset="0">
        <string>Sample</string>
       </property>
       <property name="param" stdset="0">
        <string>angles</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="DeviceParamEdit" name="orient1Edit">
       <property name="dev" stdset="0">
        <string>Sample</string>
       </property>
       <property name="param" stdset="0">
        <string>orient1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="DeviceParamEdit" name="orient2Edit">
       <property name="dev" stdset="0">
        <string>Sample</string>
       </property>
       <property name="param" stdset="0">
        <string>orient2</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Mosaic spread:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Information-only parameters:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="DeviceParamEdit" name="psi0Edit">
       <property name="dev" stdset="0">
        <string>Sample</string>
       </property>
       <property name="param" stdset="0">
        <string>psi0</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="DeviceParamEdit" name="spacegroupEdit">
       <property name="dev" stdset="0">
        <string>Sample</string>
       </property>
       <property name="param" stdset="0">
        <string>spacegroup</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="DeviceParamEdit" name="mosaicEdit">
       <property name="dev" stdset="0">
        <string>Sample</string>
       </property>
       <property name="param" stdset="0">
        <string>mosaic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>627</width>
       <height>262</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeviceParamEdit</class>
   <extends>DeviceValueEdit</extends>
   <header>nicos.guisupport.typedvalue</header>
  </customwidget>
  <customwidget>
   <class>DeviceValueEdit</class>
   <extends>QWidget</extends>
   <header>nicos.guisupport.typedvalue</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
